tbselenium/test/test_tbdriver.py

```diff
@@ -248,6 +248,9 @@ def test_font_bundling(self):
         self.assertEqual(used_font_files, bundled_font_files)
         environ["FC_DEBUG"] = ""
 
+    def test_correct_firefox_binary(self):
+        with TorBrowserDriver(TBB_PATH) as driver:
+            self.assertTrue(driver.binary.which('firefox').startswith(TBB_PATH))
```

Comment:
Iceweasel [is being renamed](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=815006) back to Firefox, so it seems safe to assume the Tor Browser Bundle won't probably use a that name for their customized binary.

Since we only care about that binary, I wouldn't check for "iceweasel" in that test.